rocky/reports/views/base.py

```diff
@@ -130,22 +130,22 @@ def get_report_types_for_aggregate_report(
     def get_required_optional_plugins(
         self, plugin_ids_dict: dict[str, set[str]]
     ) -> tuple[dict[str, list[Plugin]], dict[str, bool]]:
-        plugins = {}
-        for plugin, plugin_ids in plugin_ids_dict.items():
-            plugins[plugin] = [get_katalogus(self.organization.code).get_plugin(plugin_id) for plugin_id in plugin_ids]
+        all_plugins = {}
+        all_plugins_enabled = {}
 
-        all_plugins_enabled = {
-            plugin_type: all(plugin.enabled for plugin in plugin_list) for plugin_type, plugin_list in plugins.items()
-        }
+        for required_optional, plugin_ids in plugin_ids_dict.items():
+            plugins = []
+            are_plugins_enabled = []
+
+            for plugin_id in plugin_ids:
+                plugin = get_katalogus(self.organization.code).get_plugin(plugin_id)
+                plugins.append(plugin)
+                are_plugins_enabled.append(plugin.enabled)
 
-        return plugins, all_plugins_enabled
+            all_plugins[required_optional] = sorted(plugins, key=lambda x: x.name)
```

Comment:
Use `attrgetter` from the `operator` module
```suggestion
            all_plugins[required_optional] = sorted(plugins, key=attrgetter("name"))
```